### PR TITLE
Update build test script

### PR DESCRIPTION
**Pull request by firyx**
Updated build test script according to comments from the last PR.

### CODE REVIEW

File: build-test.sh [hunk 1]
```diff
@@ -26,40 +26,47 @@ exit_err_cleanup() {
...
-# Parameters: none
-###############################################################################
-finished() {
-  echo "$DOING_MSG finished successfully"
+  # Execute the command
+  shift 1
+	"$@"
+
+  # Log success
+	echo "Success: $MSG"
```
Comment (ro-tex) [inline build-test.sh:69]: What happens if the command fails?

File: build-test.sh [hunk 2]
```diff
@@ -106,67 +111,68 @@ wait_for_renter_upload_ready() {
...
     -f $DIR/Dockerfile \
     .
 
-  # Test with a single published port
+  # Test with a single published standard renter port
   # Run container in detached state
   DUMMY_DATA_DIR=$(mktemp -d)
-  doing "Starting test container"
-  docker run \
+  TEST1_RENTER_PORT=9980
```
Comment (ro-tex) [inline build-test.sh:134]: Do we care that this coincides with the usual port? I don't think so but it used to be 9988, so I'm asking just in case.

**Review by ro-tex — APPROVED** [review]

File: build-test.sh (hunk 1 above)
Comment (firyx) [reply build-test.sh:69]: There are settings:
```bash
## Exit immediately on error.
set -e

# Set function to be called on exit
trap 'exit_err_cleanup $?' EXIT
```

If there is an error, the script is interrupted and `exit_err_cleanup` function catches an error and exits.

The `echo "Success: $MSG"` is not executed on error.

File: build-test.sh (hunk 2 above)
Comment (firyx) [reply build-test.sh:134]: Previously there was `9988` port to check that we can forward from docker inner `9980` port to any other port.
I have updated the test script to check we can use default port and to check non-standard port.

I think it's ok to use usual port.
If somebody is affected at his localhost he can change it locally, the Travis CI is more important and there it is ok.

**Pull request merged**